.github/workflows/clang-format-linter.yml

```diff
@@ -7,9 +7,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 1
     - uses: InsightSoftwareConsortium/ITKClangFormatLinterAction@master
       with:
         itk-branch: master
+      with:
```

Comment:
Now this is duplicated. I guess this needs to be manually excluded in the remotes which already have it.

Reply:
OK, I pushed an update to remove this duplication. The CI should stay green.